app/controllers/spotlight/pages_controller.rb

```diff
@@ -149,5 +181,17 @@ def breadcrumb_to_exhibit_root(key)
     def page_params
       params.require(controller_name.singularize).permit(allowed_page_params)
     end
+
+    def redirect_page_to_related_locale_version
```

Comment:
This is how we're handling if the translated page and default language page slugs change (which is likely given that the slug is derived from the title).  Happy to look into alternatives here too.